content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+---
+For the purposes of this guide, click on the hamburger icon (to the right of the tabs) and select the "workflow: scene referred" preset now.
+
+---
+
+# image processing in 3 modules
+
+The following basic adjustments are fundamental to scene-referred editing and will be required, to some extent, on the majority of images. You can usually produce a good-looking image with these steps alone.
+
+As you will be adjusting the tones and colors of the image, start by enabling [color assessment mode](../../module-reference/utility-modules/darkroom/color-assessment.md) (press Ctrl+B) and perform the following edits on the zoomed-out image while in this mode.
+
+1. **Set overall image brightness**: First, set the overall (average) brightness of the image (the mid-gray point) by adjusting the _exposure_ slider in the [_exposure_](../../module-reference/processing-modules/exposure.md) module. This is a purely artistic setting and should be defined based on your intent -- for example, for a high-key image you will set the average brightness to be lighter than for a low-key image. The color assessment mode provides you with two reference points to assist with this by surrounding the image with a white frame against a middle-gray background.
+
+   At this point, don't worry if the brightest parts of your image lose detail -- this can be recovered in the next step.
+
+2. **Set white and black points**: The next two steps use the [_filmic rgb_](../../module-reference/processing-modules/filmic-rgb.md) module to define how the tones in your image will be mapped to the dynamic range of your display. Start by setting the _white_ and _black_ relative exposure sliders in the [_scene tab_](../../module-reference/processing-modules/filmic-rgb.md#scene). These are purely technical settings, defining white and black relative to the mid-gray point you set in the previous step. If your image contains tones you want to treat as pure white or pure black you can use the color pickers beside the sliders to set these values (using the maximum and minimum brightness of the image). Otherwise set the values manually using the color assessment frames as a reference.
+
+3. **Adjust the contrast**: Now move to the [_look tab_](../../module-reference/processing-modules/filmic-rgb.md#look) in _filmic rgb_ (for now we will skip the [_reconstruct tab_](../../module-reference/processing-modules/filmic-rgb#reconstruct)). Enable the [_look only_ view](../../module-reference/processing-modules/filmic-rgb.md#graphic-display) at the top of the module to see a representation of the filmic tone curve, which consists of a straight section in the middle (used to set the contrast of the mid-tones) and curved sections at the top and bottom (where the shadows and highlights are compressed to fit the dynamic range of the display). 
```

Comment:
Is adjusting filmic contrast something you really need do for every image? It can be mentioned as a tool available if needed, but tone equaliser is probably more important where tones in certain parts of the picture need to be adjusted.

Reply:
I often don't accept the defaults. I think adjusting contrast is a common enough operation to include it here.